src/components/coinPriceChange/content/CoinPriceChange.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { PriceChangeContainer } from "../style/coinPriceChange_style";
+import { CoinPriceChangeProps } from "./coinPriceChange_types";
+
+export const CoinPriceChangeSection: React.FC<CoinPriceChangeProps> = ({
+  priceChange,
+}) => {
+  if (priceChange === 0)
+    return (
+      <PriceChangeContainer priceChange={priceChange}>0</PriceChangeContainer>
+    );
+  return priceChange < 0 ? (
```

Comment:
no duplicate code is allowed

Reply:
resolved , although we removed this section after littleLineChart added instead